README.md

```diff
@@ -2,15 +2,36 @@
 
 QGIS plugin for drone image classification using machine learning.
 
-## Installation
+## Installation (prototype)
 
-Clone this GitHub repository to your local machine.
+Dowload the [ZIP file of DroneML](https://nlesc-my.sharepoint.com/:u:/g/personal/o_ku_esciencecenter_nl/ETVp8wcBhJ1Com3PDohe5eQB219nCHs6wJURJymwqR6uRw?e=lOYkvn).
+
+In QGIS, go to `Plugins` > `Python Console`.
+
+Run the following Python code to install the dependencies required by the plugin:
+
+```python
+import pip
+pip.main(["install", "tqdm", "rioxarray", "geopandas", "scikit-learn", "torch", "torchvision", "torchinfo"]
+```
+
```

Comment:
```suggestion
Restart QGIS for changes to your Python package installation to take effect.
```

I suppose that somehow restarting python would work as well, but I don't know how to do that so I only was able to solve it by restarting QGIS completely. Otherwise, it would keep crashing on the missing 'rioxarray' etc.